e2e/cypress/integration/team_settings/teams_spec.js

```diff
@@ -211,4 +216,176 @@ describe('Teams Suite', () => {
+
+        // Save new team description
+        cy.get('#saveSetting').click();
+
+        // # Close the team settings
+        cy.get('body').type('{esc}', {force: true});
+
+        // # Functionality
+
+        // # Require license and create new user
+        cy.apiRequireLicenseForFeature('GuestAccounts');
+
+        cy.visit('/login');
+        cy.apiLogin(newUser);
+
+        // * Verify if the user is redirected to the Select Team page
+        cy.url().should('include', '/select_team');
+        cy.get('#teamsYouCanJoinContent').should('be.visible');
+        cy.wait(TIMEOUTS.ONE_SEC);
+
+        // # Select test team name
+        cy.get(`#${testTeam.name.charAt(0).toUpperCase() + testTeam.name.slice(1)}`).click();
+        cy.wait(TIMEOUTS.THREE_SEC);
+
+        // # Verify Town square is visible
+        cy.url().should('include', `/${testTeam.name}/channels/town-square`);
+        cy.findByText('Town Square').should('be.visible');
+    });
+
+    it('MM-T2322/MM-T2323 Team setting / Do not allow anyone to join this team', () => {
+        // # UI
+
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Open the hamburger menu
+        cy.findByLabelText('main menu').should('be.visible').click();
+
+        // # Click on team settings menu item
+        cy.findByText('Team Settings').should('be.visible').click();
+
+        // # Click on the team description menu item
+        cy.findByText('Allow any user with an account on this server to join this team').should('be.visible').click();
+
+        // # Change team description in the input
+        cy.get('#teamOpenInviteNo').click();
+
+        // Save new team description
+        cy.get('#saveSetting').click();
+
+        // # Close the team settings
+        cy.get('body').type('{esc}', {force: true});
+
+        // # Functionality
+
+        // # Require license and login new user
+        cy.apiRequireLicenseForFeature('GuestAccounts');
+        cy.apiLogout();
+        cy.wait(TIMEOUTS.ONE_SEC);
+
+        cy.visit('/login');
+        cy.apiLogin(newUser);
+
+        // * Verify if the user is redirected to the Select Team page
+        cy.url().should('include', '/select_team');
+        cy.get('#teamsYouCanJoinContent').should('be.visible');
+        cy.wait(TIMEOUTS.ONE_SEC);
+
+        // * Verify the user can login to a team
+        cy.get('.signup-team-dir').children().first().click();
+        ignoreUncaughtException();
+
+        // # Open the hamburger menu
+        cy.findByLabelText('main menu').should('be.visible').click();
+
+        // # Click on Join another team menu item
+        cy.findByText('Join Another Team').should('be.visible').click();
+
+        // # Verify the original test team isn't on the lit
+        cy.get('.signup-team-dir').children().should('not.contain', `#${testTeam.name.charAt(0).toUpperCase() + testTeam.name.slice(1)}`);
+    });
+
+    it('MM-T2328 Team setting / Member can view and search for members', () => {
```

Comment:
nit: Please match the title. Also, we can update the TM4J title if you feel it should be the case (very much open to suggestion).
```suggestion
    it('MM-T2328 Member can view and search for members', () => {
```